integration_test/gce/gce_testing.go

```diff
@@ -803,7 +822,54 @@ func RunRemotely(ctx context.Context, logger *log.Logger, vm *VM, stdin string,
 	args = append(args, "-oIdentityFile="+privateKeyFile)
 	args = append(args, sshOptions...)
 	args = append(args, wrappedCommand)
-	return runCommand(ctx, logger, stdin, args)
+	
+	output, exitCode, err := runCommand(ctx, stdin, args)
+
+	if IsWindows(vm.Platform) {
+		// This is a workaround for the fact that powershell
+		// ignores -OutputFormat Text in certain situations (fixed in pwsh but
+		// not powershell): https://github.com/PowerShell/PowerShell/issues/5912
+		output.Stderr = stderrFromXML(output.Stderr)
+	
+		if err != nil {
+			err = fmt.Errorf("Command failed with exit code %v: %v\n%v\nstdout: %s\nstderr: %s", exitCode, command, err, output.Stdout, output.Stderr)
+		}
+		logger.Printf("exit code: %v", exitCode)
+		logger.Printf("stdout: %s", output.Stdout)
+		logger.Printf("stderr: %s", output.Stderr)
+	} else {
+		if err != nil {
+			err = fmt.Errorf("Command failed with exit code %v: %v\n%v\nstdout+stderr: %s", exitCode, command, err, output.Interleaved)
+		}
+		logger.Printf("exit code: %v", exitCode)
+		logger.Printf("stdout+stderr: %s", output.Interleaved)
+	}
+
+	return output, err
+}
+
+// stderrFromXML extracts the human-readable part of the XML returned by
+// powershell over stderr.
```

Comment:
Maybe we should clarify here, that if its not XML already (in case the powershell bug is fixed later), this returns the raw stderr